### ISSUE TITLE
Why is zero-loss-cycle always part of loss computation?

### ISSUE THREAD

**Issue by vadimkantorov** [ajabri/videowalk#34]
https://github.com/ajabri/videowalk/blob/master/code/model.py#L171 :
`xents = [torch.tensor([0.]).to(self.args.device)]`
`loss = sum(xents)/max(1, len(xents)-1)`

Is it just to have something evaluated to zero if there are no walks? In what case does this happen in practice?

Thank you @ajabri !

**Comment by ajabri** [comment]
Hi @vadimkantorov,

Yes, I believe that's the reason for the line, but it can be removed!
